tests/Behat/Context/Ui/Shop/CheckoutContext.php

```diff
@@ -14,124 +14,24 @@
 
 use Behat\Behat\Context\Context;
 use Behat\MinkExtension\Context\RawMinkContext;
-use Sylius\Behat\Page\Shop\Order\ShowPageInterface;
-use Tests\BitBag\SyliusMolliePlugin\Behat\Mocker\MollieApiMocker;
-use Tests\BitBag\SyliusMolliePlugin\Behat\Page\External\PaymentPageInterface;
 use Tests\BitBag\SyliusMolliePlugin\Behat\Page\Shop\Checkout\CompletePageInterface;
 
 final class CheckoutContext extends RawMinkContext implements Context
 {
     /** @var CompletePageInterface */
     private $summaryPage;
 
-    /** @var ShowPageInterface */
```

Comment:
Why so much code is being removed?

Reply:
Functions were depending on outdated mockery